jinahub/encoders/text/CLIPTextEncoder/README.md

```diff
@@ -1,9 +1,57 @@
 # CLIPTextEncoder
 
- **CLIPTextEncoder** is a class that wraps the text embedding functionality using the **CLIP** model from huggingface transformers
+**CLIPTextEncoder** is a text encoder that wraps the text embedding functionality using the **CLIP** model from huggingface transformers.
 
-The **CLIP** model was originally proposed in  [Learning Transferable Visual Models From Natural Language Supervision](https://arxiv.org/abs/2103.00020).
+It takes `Document`s with text stored in the `text` attribute as inputs, and stores the
+resulting image embedding in the `embedding` attribute.
 
+The **CLIP** model was originally proposed in [Learning Transferable Visual Models From Natural Language Supervision](https://arxiv.org/abs/2103.00020), and is trained to embedd images and text to the same latent
+spance. The corresponding image encoder is **[CLIPImageEncoder](https://hub.jina.ai/executor/0hnlmu3q)**,
+using both encoders together works well in multi-modal or cross-modal search applications.
+
+## Usage
+
+Here's a simple example of how to use CLIPTextEncoder in a Flow
+
+```python
+from jina import Flow, Document
+
+f = Flow().add(uses='jinahub+docker://CLIPTextEncoder')
+
+def print_result(resp):
+    doc = resp[0].docs[0]
+    print(f'Embedded "{doc.text}" to {doc.embedding.shape[0]}-dimensional vector')
+
+with f:
+    doc = Document(text='your text')
+    f.post(on='/foo', inputs=doc, on_done=print_result)
+```
```

Comment:
when I test this I receive:
```
  File "/home/ec2-user/jina/test-executors/venv/lib/python3.7/site-packages/jina/clients/helper.py", line 91, in callback_exec
    _safe_callback(on_done, continue_on_error, logger)(response)
  File "/home/ec2-user/jina/test-executors/venv/lib/python3.7/site-packages/jina/clients/helper.py", line 66, in _arg_wrapper
    raise BadClientCallback(err_msg) from ex
jina.excepts.BadClientCallback: uncaught exception in callback print_result(): TypeError("'Response' object is not subscriptable")
```

I believe this can be a fix:

````suggestion
```python
from jina import Flow, Document

f = Flow().add(uses='jinahub+docker://CLIPTextEncoder')

def print_result(resp):
    doc = resp.docs[0]
    print(f'Embedded "{doc.text}" to {doc.embedding.shape[0]}-dimensional vector')

with f:
    doc = Document(text='your text')
    f.post(on='/foo', inputs=doc, on_done=print_result)
```
````

Reply:
Also same thing probably with clip image encoder please double check